assets/css/screen.css

```diff
@@ -1213,6 +1341,162 @@ Usage (In Ghost editor):
     text-decoration: none;
 }
 
+.authors {
+    display: flex;
+    flex-grow: 1;
+    flex-direction: column;
+    align-items: center;
+    border-top: color(var(--lightgrey) l(+10%)) 1px solid;
+    padding-top: 40px;
+    margin-top: 20px;
+}
+
+.authors .names p {
+    font-size: 1.4rem;
+    letter-spacing: 0.2px;
+    color: var(--midgrey);
+    text-align: center;
+    text-transform: uppercase;
+    margin-bottom: 0;
```

Comment:
Need to review property orders - margin should be at the top of this list. Maybe try CSScomb?